### ISSUE TITLE
Correct FAQ entry for green/low risk encounters

### ISSUE THREAD

**Issue by daimpi** [corona-warn-app/cwa-website#307]
## Where to find the issue
[German](https://www.coronawarn.app/de/faq/#encounter_but_green) and [English](https://www.coronawarn.app/en/faq/#encounter_but_green) FAQ entry on green risk encounters.

## Describe the issue
~~The current entry states:~~
~~> All exposures for a diagnosis key that lasted less than 10 minutes in total (regardless of how close the smartphones came during that time) or during which the smartphones were more than 8 meters (73 dB attenuation) apart on average (regardless of how long the exposure lasted) are discarded as harmless. This is why the Corona-Warn-App can display encounters, but the risk status stays the same.~~

~~This suggests that encounters which are "discarded as harmless" are actually shown as green-/low-risk encounters, which is [not](https://github.com/corona-warn-app/cwa-backlog/issues/23#issuecomment-678644207) [correct](https://github.com/corona-warn-app/cwa-backlog/issues/23#issuecomment-678273003).~~

~~Also: This suggests that only encounters shorter than 10 minutes would be discarded (which I guess corresponds to discarding encounters which only appeared in **one** listening window), when in reality encounters that are shorter than 15 min (i.e. up to and including 10 min ([`gt_5_le_10_min: 0`](https://github.com/corona-warn-app/cwa-server/blob/8c2fea20c9e234db0ec5bda038296ecda7adee2f/services/distribution/src/main/resources/master-config/exposure-config.yaml#L33)) / **two** listening windows) are discarded.~~

## Suggested change
~~1. Change the text so that it is clear that such encounters will be hidden from the user and not shown as green-/low-risk encounters.
2. Change the text regarding the time cutoff to state that encounters shorter than 15 min are discarded.~~

**Edit**: ~~The 2nd point actually also applies to the risk-assessment documentation which is linked there and [states](https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md#procedure):~~ 
~~> All encounters for a diagnosis key that lasted less than 10 minutes in total […] are discarded as negligible risk.~~

~~This should also say "less than 15 minutes".~~

**Edit2**: ~~Regarding the change to "less than 15 minutes": just make sure that I'm not talking nonsense here 😅 
After looking at the config file again and seeing the [` eq_0_min`](https://github.com/corona-warn-app/cwa-server/blob/8c2fea20c9e234db0ec5bda038296ecda7adee2f/services/distribution/src/main/resources/master-config/exposure-config.yaml#L31) entry (which seems to be pretty nonsensical if this would actually correspond to **zero** listening windows) I'm not entirely sure how the time mapping works, so just check with the ppl who know the details on how exactly this mapping works before changing anything.~~

---

**Edit3**: ~~As @kbobrowski already stated below: it turns out that the FAQ entry seems to be correct after all: https://github.com/corona-warn-app/cwa-backlog/issues/23#issuecomment-679231707~~

~~I'd suggest putting this issue on hold until this has been clarified by the devs, just to be sure. If they confirm this, the issue can be closed. Although the question on how exactly the mapping/interaction between duration and exposure windows works would still be interesting 🙂.~~ 

**Edit4**: After https://github.com/corona-warn-app/cwa-website/pull/343 has been merged, the FAQ now actually contains a bug: the new formulation suggests that not all encounters would be shown in the app which is contradicted by @kbobrowski's [findings](https://github.com/corona-warn-app/cwa-website/issues/307#issuecomment-684774771). I would suggest reverting #343 until this issue is sorted out to reduce the number of people receiving misinformation via the FAQ but in the meantime you can replace the enhancement label of this issue with the bug label again.

---
Internal Tracking ID: EXPOSUREAPP-2344

**Comment by kbobrowski** [comment]
I have corrected my experiment after accounting for certain sanity check inside EN: https://github.com/corona-warn-app/cwa-backlog/issues/23#issuecomment-678644207, and it seems that all encounters, regardless of how CWA is configured, would be displayed in log and as "low-risk exposure" in CWA, so I think the FAQ entry is correct

**Comment by Ein-Tim** [comment]
Even if the FAQ entry is correct, I think it should be made clearer that all Encounters are shown to the user.
In English it says:

"All exposures for a diagnosis key that lasted less than 10 minutes in total [...] or during which the smartphones were more than 8 meters [...] apart on average [...] are **discarded** as harmless."
But then there is written: 
"This is why the Corona-Warn-App can display encounters, but the risk status stays the same."

In German the same:

"Alle Begegnungen zu einer Positivkennung, die insgesamt weniger als 10 Minuten gedauert haben [...] oder bei denen die Smartphones im Durchschnitt mehr als ca. 8 Meter Freiraum [...] voneinander entfernt waren [...], werden als unbedenklich **verworfen**. Deswegen kann die Corona-Warn-App Ihnen Begegnungen gemeldet haben, ohne dass die Risikobewertung sich ändert.

The word "discarded"(="verworfen") implicate that there are some Exposures who are discarded.
The user can misunderstand it and think not everything is shown to him/her. 
Better would be:
"[...] are rated as harmless[...]"
("[...] als unbedenklich bewertet [...]")

**Comment by svengabr** [comment]
Hello everyone,

thank you all for your feedback!

Regarding the initial concern of @daimpi it seems that 10 minutes are indeed correct.
The latest configuration can always be found here:
https://github.com/corona-warn-app/cwa-server/blob/master/services/distribution/src/main/resources/master-config/exposure-config.yaml

---

Regarding the redactional change that @Ein-Tim suggested, I will now forward this to the responsible person who can do this change for us. I think it's a good improvement and will lead to less confusion.

Best regards,
SG

Corona-Warn-App Open Source Team

**Comment by daimpi** [comment]
My main motivation for opening this issue was actually the first point, regarding the suggestion that "discarded encounters will be shown as green-/low-risk encounters". This was a contradiction to @thomasaugsten's comment [here](https://github.com/corona-warn-app/cwa-backlog/issues/23#issuecomment-678273003) which stated that such encounters would not be shown in either CWA nor the ENF log in any form. But since @kbobrowski has tested this and found that actually no encounters will be hidden from the user, this contradiction no longer exists (even though it would be nice to actually get confirmation on this by the devs, just to be sure). 

I also agree with @Ein-Tim's statement above, that using the word "discarded" seems misleading and should be improved. When improving the FAQ entry I'd also suggest improving those statements in the [risk-assessment documentation](https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md) which the FAQ entry links to, to make it clear that no encounters will be hidden from the user.

Regarding the 2nd point (the "less than 10 min" statement): after [reading](https://developers.google.com/android/exposure-notifications/exposure-notifications-api) a bit more, it seems that listening windows are not necessarily 5 min apart, but rather that there also exist closer scan intervals of only 2.5 min apart. 
Given this and @mh-'s [statement on Gitter](https://gitter.im/coronawarn/community?at=5f40047addc2d041c0baa515), that timestamps of the scan windows are used, I think this point has indeed become moot.

**Comment by tkowark** [comment]
Could you kindly check #307 to see whether that suffices for this issue? @daimpi @Ein-Tim

**Comment by Ein-Tim** [comment]
Looks good to me 👍

**Comment by tkowark** [comment]
Oops, that link was supposed to be #319 ... my bad

**Comment by daimpi** [comment]
Looks good to me 👍
I've added an additional suggestion in the German diff: https://github.com/corona-warn-app/cwa-website/pull/319#discussion_r477273800

What do you think wrt the [risk-assessment documentation](https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md) which the FAQ entry links to? Should we make a separate issue in the documentation repo for this? 🙂

**Comment by daimpi** [comment]
@tkowark could you maybe confirm with the CWA iOS devs/Apple whether @kbobrowski's findings (i.e. no encounter will be hidden in CWA & the ENF log) also apply to iOS, just to be save? 🙂

**Comment by Ein-Tim** [comment]
@tkowark 
Could you please make clearer if there are any encounters **hidden** from the user, like it says in [this "Zeit-Artikel"](https://www.zeit.de/digital/2020-08/corona-warn-app-coronavirus-eindaemmung-karl-lauterbach-henning-tillmann).
Thank you 👍

**Comment by thomasaugsten** [comment]
This are the encounters which are filtered out by the ENF via the configuration. This encounters are not sent to the CWA and not count in the matchCount.

**Comment by Ein-Tim** [comment]
Ok, and these are the Encounters under 10 Minutes and not _nearer_ than 73 db, right?

**Comment by daimpi** [comment]
@thomasaugsten 
>This are the encounters which are filtered out by the ENF via the configuration. This encounters are not sent to the CWA and not count in the matchCount.

your comment is exclusively about iOS? B/c under Android obviously **no** encounters are filtered out. Every encounter is [at least shown](https://github.com/corona-warn-app/cwa-website/issues/307#issuecomment-679233521) as "low risk/green". Could you please clarify?

**Comment by kbobrowski** [comment]
As far as we could confirm, on Android all the encounters are displayed at least as "low-risk encounter", even if it lasted 5 minutes with 100dB attenuation, this was also independently confirmed: https://covidapps.slack.com/archives/C0194ML0MLN/p1598517214083800

**Comment by thomasaugsten** [comment]
iOS and Android using the same configuration for the ENF
https://github.com/corona-warn-app/cwa-app-android/blob/024cb66d37a21ae1b29643b5bdca5f3796cb7973/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/transaction/RiskLevelTransaction.kt#L219

If you see a different behavior of the Android ENF please a create ticket with detailed information we will check this with google.

**Comment by daimpi** [comment]
@thomasaugsten : Are you saying that the behavior seen under Android wrt those "very low risk" encounters is not the same as you observe under iOS? Were you able to test this? B/c according to [Google spec](https://developers.google.com/android/exposure-notifications/exposure-notifications-api) 

![grafik](https://user-images.githubusercontent.com/22129968/91845733-f3844d80-ec59-11ea-8c76-785cafe79ccd.png)

In particular this last sentence:
>Other returned fields are unaffected by this setting.

it seems like `ExposureSummary` not being affected by `minimumRiskScore` is the **intended behavior**. This is also coherent with @kbobrowski's findings. Is this different under iOS?

**Comment by kbobrowski** [comment]
@thomasaugsten I cannot verify what is the behavior on iOS, can only report here that on Android no encounters are "filtered out" inside ENF by configuration - meaning that all the encounters, however short or weak signal, would be included in "matchedKeyCount". You can see it in this comment: https://github.com/corona-warn-app/cwa-backlog/issues/23#issuecomment-679231707 , encounter had very weak signal (above 80), but still ENF counted it in "matchedKeyCount", although the risk value associated with this encounter is 0. I'm not sure if this is the issue as it seems to be intended behavior.

**Comment by thomasaugsten** [comment]
Thanks for the hint we will double check this with google.

**Comment by daimpi** [comment]
@thomasaugsten 
> Thanks for the hint we will double check this with google.

Thanks 🙂 
Could you also in parallel check with Apple just to be save?

**Comment by thomasaugsten** [comment]
Yes of course we will check this with apple to ensure we have a consistent API here.

**Comment by daimpi** [comment]
I've edited the [OP](https://github.com/corona-warn-app/cwa-website/issues/307#issue-684024621) (Edit4) to reflect the new development that came from merging #343. I'd suggest replacing the *enhancement* label of this issue with the *bug* label again, as the FAQ unfortunately contains misleading information now.

**Comment by svengabr** [comment]
We got an official statement from the Robert Koch Institute:

**German**
>Als Herausgeber der Corona-Warn-App bedankt sich das Robert Koch-Institut für die vielen Rückmeldungen zur Risikostatusanzeige auf dem HomeScreen. Diesem Dank schließen wir uns als Entwicklerteam der App an. Nach Abwägung aller Argumente und in Abstimmung mit dem Bundesministerium für Gesundheit hat das RKI entschieden, die zweistufige Risikodarstellung (grün und rot) beizubehalten.

>Die App muss möglichst einfach und allgemeinverständlich in der Anwendung sein. Durch die Unterscheidung von niedrigem und erhöhtem Risiko können den Nutzerinnen und Nutzern auf ihren Risikostatus abgestimmte Handlungsempfehlungen gegeben werden. Die zusätzliche Anzeige von Begegnungen, die bestimmte Kriterien erfüllen, erhöht die Transparenz. Für alle diejenigen, die sich mit der Risikobewertung und ihren Parametern eingehender beschäftigen möchten, haben RKI und Entwicklerteam detaillierte Beschreibungen auf GitHub zur Verfügung gestellt.

>Die Texte in der Corona-Warn-App werden wir unter Berücksichtigung Ihres Feedbacks weiter anpassen. Mit einem anstehenden Update werden wir weitere Erläuterungen zum besseren Verständnis der angezeigten Risikobegegnungen geben. Zudem wird geprüft, ob mit einem späteren Update eine Risikobegegnungshistorie in der App dargestellt werden kann.

>Auch zukünftig werden wir Anregungen und Vorschläge zur Verbesserung der Corona-Warn-App aufgreifen, diskutieren und, soweit sinnvoll und möglich, umsetzen. Wir freuen uns, wenn Sie sich weiter mit konstruktivem Feedback beteiligen.

---

**English**
>As the publisher of the Corona-Warn-App, the Robert Koch-Institute as well as the development team would like to thank you for the many feedbacks on the risk status display on the HomeScreen. After weighing up all arguments and in coordination with the Federal Ministry of Health, the RKI has decided to keep the two-level risk display (green and red).

>The app must be as simple and generally understandable as possible in its application. By differentiating between low and increased risk, users can be given recommendations for action based on their risk status. The additional display of encounters that meet certain criteria increases transparency. For those who want to take a closer look at risk assessment and its parameters, RKI and the development team have provided detailed descriptions on GitHub.

>We will further improve the texts in the Corona-Warn-App and we will continue to listen for feedback of the community. With an upcoming update, we will provide further explanations to help you better understand the risk encounters displayed. In addition, we will check whether a later update will allow a risk encounter history to be displayed in the app.

>In the future, we will continue to take up, discuss and, where reasonable and possible, implement suggestions and proposals for improving the Corona Warning App. We would be pleased if you continue to contribute with constructive feedback.

Further text improvements are being introduced in the upcoming hotfix release 1.3.1.

RC1 of 1.3.1 is already available:
https://github.com/corona-warn-app/cwa-app-android/releases/tag/1.3.1-SNAPSHOT-RC1
https://github.com/corona-warn-app/cwa-app-ios/releases/tag/v1.3.1-RC1

Since the decision from the RKI is final, I will close this issue now.

Best regards,
SG

Corona-Warn-App Open Source Team

**Comment by daimpi** [comment]
Hi @thomasaugsten 
> Yes of course we will check this with apple to ensure we have a consistent API here.

Were you able to confirm that this behavior is consistent between Android and iOS (i.e. that all encounters are shown on both OS)?
